.github/workflows/kernel-test.yml

```diff
@@ -135,30 +157,111 @@ jobs:
             }
 
             routing {
-                dip(1.1.1.1) -> proxy
-                fallback: direct
+                pname(dae) -> direct
```

Comment:
Why we need this?

Reply:
https://github.com/daeuniverse/dae/issues/384

Pid matching doesnt work inside container